index.html

```diff
@@ -0,0 +1,286 @@
+                fill="#FF6363"
+              />
+            </svg>
+            <p class="steps__text">Select Places</p>
+          </div>
+          <div class="steps__icon-box">
+            <svg
+              width="96"
+              height="96"
+              viewBox="0 0 96 96"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <rect width="96" height="96" rx="48" fill="#333333" />
+              <path
+                d="M74.2837 51.8363C75.38 49.98 76 48 76 45.6063C76 40.1044 71.3477 34.9088 65.2725 34.9088H60.7125C61.3275 33.3075 61.8187 31.3925 61.8187 29.0913C61.8187 19.992 57.1075 16 49.91 16C42.2091 16 42.6484 27.8666 40.94 29.575C38.0966 32.4184 34.7381 37.8809 32.345 40H20C17.7909 40 16 41.7909 16 44V74C16 76.2091 17.7909 78 20 78H28C29.8616 78 31.426 76.7282 31.8722 75.0062C37.4359 75.1314 41.2547 79.9987 54.0975 79.9987C55 79.9987 56 80 56.875 80C66.5146 80 70.8732 75.0721 70.9925 68.0838C72.6574 65.7806 73.5299 62.6935 73.16 59.71C74.3918 57.4035 74.868 54.6671 74.2837 51.8363ZM66.565 58.565C68.135 61.2062 66.7225 64.7412 64.8225 65.7612C65.785 71.8587 62.6215 73.9987 58.1825 73.9987H53.455C44.5001 73.9987 38.7014 69.2713 32 69.2713V46H33.365C36.91 46 41.8625 37.1387 45.1825 33.8175C48.7275 30.2725 47.5462 24.3638 49.91 22C55.8187 22 55.8187 26.1225 55.8187 29.0913C55.8187 33.9875 52.2738 36.1813 52.2738 40.9088H65.2725C67.9112 40.9088 69.9888 43.2725 70 45.6362C70.0112 47.9987 68.3975 50.3625 67.2163 50.3625C68.9024 52.1819 69.2626 56.017 66.565 58.565ZM27 70C27 71.6569 25.6569 73 24 73C22.3431 73 21 71.6569 21 70C21 68.3431 22.3431 67 24 67C25.6569 67 27 68.3431 27 70Z"
+                fill="#00A2FF"
+              />
+            </svg>
+            <p class="steps__text">Book & Enjoy</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- and section 2 -->
+    <!-- start section 3 -->
+    <section class="booknow">
+      <div class="container">
+        <div class="booknow__inner">
+          <div class="booknow__card">
+            <p class="booknow__subtitle">Spain</p>
+            <h4 class="booknow__title">Grand Spain Madrid</h4>
+            <p class="booknow__text">
+              But I must explain to you how all this mistaken idea of account to
+              denouncing pleasure and praising pain was account born and I
+              account will give you a complete account of the system
+            </p>
+            <a href="#" class="button">book now </a>
+          </div>
```

Comment:
я бы даже выделил это в отдельный блок `.card`
```suggestion
          <div class="card booknow__card">
            <p class="card__subtitle">Spain</p>
            <h4 class="card__title">Grand Spain Madrid</h4>
            <p class="card__text">
              But I must explain to you how all this mistaken idea of account to
              denouncing pleasure and praising pain was account born and I
              account will give you a complete account of the system
            </p>
            <a href="#" class="button card__button">book now </a>
          </div>
```